ocs_ci/utility/pagerduty.py

```diff
@@ -247,15 +287,21 @@ def get_service_dict(self):
         cluster_name = config.ENV_DATA["cluster_name"]
         # timestamp is added to service name to ensure unique name of service
         timestamp = time.time()
-        default_policy = self.get_default_escalation_policy_id()
+        policy = None
+        policy_name = config.AUTH["pagerduty"].get("escalation_policy")
+        if policy_name:
```

Comment:
And if this condition is not satisfied, would it make sense to raise some meaningful error?

Reply:
Indentation updated so that ValueError is raised even when the value is None